src/libs/lodash.ts

```diff
@@ -1,5 +1,5 @@
-import _merge = require('lodash.merge')
-import _range = require('lodash.range')
+import _merge from 'lodash.merge'
+import _range from 'lodash.range'
```

Comment:
Although it is unreliable memory, this description used `require` because a runtime error occurs on the side using smarthr-ui.
So if you use `from`, you will need to make sure there is no problem with runtime.

Reply:
@nabeliwo @im36-123 
> Although it is unreliable memory, this description used require because a runtime error occurs on the side using smarthr-ui.

Yes, importing a CommonJS module with `import a from 'a'` statement isn't recognized by default on TypeScript.
But It works well at this PR because I've enabled `esModuleInterop`  flag when introducing Jest.
(I've tested this build on a project created by `create-react-app`)

https://github.com/kufu/smarthr-ui/commit/6d9d95f7000d9dff73fe77b396276c35f9c65d26#diff-e5e546dd2eb0351f813d63d1b39dbc48

LGTM 👍

Reply:
Thanks!